src/bullsAndCows.js

```diff
@@ -26,7 +26,28 @@
  * @return {object} - like {bulls: 0, cows: 0}
  */
 function bullsAndCows(generatedNumber, enteredNumber) {
-  // write code here
+  const gameScore = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  if (enteredNumber === undefined || enteredNumber.length !== 4) {
+    return undefined;
+  }
+
+  for (let i = 0; i < enteredNumber.length; i++) {
+    if (enteredNumber.lastIndexOf(enteredNumber[i]) !== i) {
```

Comment:
what are you trying to do here?

Reply:
I'm trying to return undefined if values in 'enteredNumber' aren't unique